packages/react-core/src/components/NumberInput/examples/NumberInputUnitThreshold.tsx

```diff
@@ -24,8 +24,17 @@ export const NumberInputUnitThreshold: React.FunctionComponent = () => {
 
   const onChange = (event: React.FormEvent<HTMLInputElement>) => {
     const target = event.target as HTMLInputElement;
-    const newValue = normalizeBetween(isNaN(+target.value) ? 0 : Number(target.value), minValue, maxValue);
-    setValue(newValue);
+    setValue(+target.value);
```

Comment:
The issues asks "Allow the user to clear the value using the Delete key. This means that they will see no value displayed in the input."

The "+" here wouldn't allow for that. As this is the example you are working with for this change, maybe this could be something like:
```
const value = (event.target as HTMLInputElement).value;
setValue(value === "" ? value : +value);
```

Reply:
Should the value type (NumberInput.tsx:13) also include empty string?